### DISCUSSION TITLE
Incoming Connections

### DISCUSSION THREAD

**Discussion by huwpne** [ViennaRSS/vienna-rss#1450] [Q&A] [answered]
Can you advise if Vienna needs Incoming Connections enabled within the Apple Inbuilt Firewall

Many thanks

**Comment by Eitot** [comment]
I do not think that Vienna needs incoming connections. The internal browser may be using it, but I cannot say for which purpose.

**Reply by huwpne** [reply, marked as answer]
Thanks, I disabled it and have had no issues since
